### ISSUE TITLE
Estalagem

### ISSUE THREAD

**Issue by Welliton83** [Welliton83/JvS#1]
<!DOCTYPE html>
 <html lang="en">
 <head>
     <title>Cadastro do Viajante</title>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 </head>
 <body>
     <div class="col-12 col-md-9 col-xl-8 py-md-3 pl-md-5 bd-content">
 
   
     <h3>Estalagem Dalaran</h1>
     <h4>Cadastro de Viajante </h2>
     <hr>
     <form>
         <div class="form-group">
             <button type="button" class="btn btn-info">Allyance</button>
             <button type="button" class="btn btn-info">Horde</button>
             
         </div>
         <div class="form-group">
             <label for="input1">Email</label>
             <input type="email" class="form-control" id="input1">
         </div>
         <div class="form-group">
             <label for="input2">Senha</label>
             <input type="email" class="form-control" id="input2" >
         </div>
         
         <div class="form-group">
             <label for="input4">Apartamento</label>
             <input type="email" class="form-control" id="input4">
         </div>
         <div class="form-group">
             <label for="input5">Número</label>
             <input type="email" class="form-control" id="input5">
         </div>
         <div class="form-group">
             <label for="input6">Caixa de correio</label>
             <input type="email" class="form-control" id="input6">
         </div>
         <div class="form-group">
             <label for="input7">Possui Montaria?</label>
             <select class="form-control" id="input7">
                 <option value="s">Sim</option>
                 <option value="n" selected >Nao</option>
             </select>
         </div>
         <div class="info_carro">
             <div class="form-group">
                 <label for="input8">Nome da Montaria</label>
                 <input type="email" class="form-control" id="input8">
             </div>
             <div class="form-group">
                 <label for="input9">Raça</label>
                 <input type="email" class="form-control" id="input9">
             </div>
               <div class="form-group">
             <label for="input10">Terrestre ou Voadora</label>
             <select class="form-control" id="input7">
                 <option value="s">Terrestre</option>
                 <option value="n" selected >Voadora</option>
             </select>
         </div>
         </div>
         
             
         <button type="submit" class="btn btn-primary">Entrar</button>
         <button type="reset" class="btn btn-secondary">Limpar</button>
         
        
         <button type= "button" class="btn btn- info" onclick="myFunction()">Salvar</button>

<script>
function myFunction() {
    alert("Salvo com Sucesso!");
}
</script>

<button type= "button" class="btn btn- info" onclick="myFunction()">Excluir</button>

<script>
function myFunction() {
    confirm("O conteudo foi Excluido!");
}
</script>
     </form>
     </body>
 
 
 </div>
 <script>
 
 $(function(){
 
     $(".info_carro").hide();
 
     $("#input7").on("change", function(){
         if( $(this).val() == 's'){
             $(".info_carro").show();
         }else{
             $(".info_carro").hide();
         }
         
     })
 })
 </script>
 </body>
 <!DOCTYPE html>
 <html lang="en">
 <head>
     <title>Cadastro do Viajante</title>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 </head>
 <body>
     <div class="col-12 col-md-9 col-xl-8 py-md-3 pl-md-5 bd-content">
 
   
     <h3>Estalagem Dalaran</h1>
     <h4>Cadastro de Viajante </h2>
     <hr>
     <form onsubmit="return validate(this);">
         <div class="form-group">
             <button type="button" class="btn btn-info">Allyance</button>
             <button type="button" class="btn btn-info">Horde</button>
         </div>
         <div class="form-group">
             <label for="input1">Email</label>
             <input type="email" required class="form-control" name="input1" id="input1">
         </div>
         <div class="form-group">
             <label for="input2">Senha</label>
             <input type="password" required class="form-control" name="input2" id="input2" >
         </div>
 
         <div class="form-group">
             <label for="input4">Apartamento</label>
             <input type="text" required class="form-control" name="input4" id="input4">
         </div>
         <div class="form-group">
             <label for="input5">Número</label>
             <input type="number" required class="form-control" name="input5" id="input5">
         </div>
         <div class="form-group">
             <label for="input6">caixa de correio</label>
             <input type="tel" required class="form-control" name="input6" id="input6">
         </div>
         <div class="form-group">
             <label for="input22">Possui Montaria?</label>
             <select class="form-control" id="input22">
                 <option value="s">Sim</option>
                 <option value="n" selected >Nao</option>
             </select>
         </div>
         <div class="info_montaria">
             <div class="form-group">
                 <label for="input8">Nome da Montaria</label>
                 <input type="email" class="form-control" id="input8">
             </div>
             <div class="form-group">
                 <label for="input9">Raça</label>
                 <input type="email" class="form-control" id="input9">
             </div>
             <div class="form-group">
             <label for="input10">Terrestre ou Voadora</label>
             <select class="form-control" id="input22">
                 <option value="s">Terrestre</option>
                 <option value="n" selected >Voadora</option>
             </select>
         </div>
         </div>
         
             
         <button type="submit" class="btn btn-primary">Entrar</button>
         <button type="reset" class="btn btn-secondary">Limpar</button>
         
         <button type= "button" class="btn btn- info" onclick="myFunction()">Salvar</button>

<script>
function myFunction() {
    alert("Salvo com Sucesso!");
}
</script>

    <button type= "button" class="btn btn- info" 		 onclick="myFunction()">Excluir</button>

    <script>
    function myFunction() {
        confirm("O conteudo foi Excluido!");
    }
    </script>

     </form>
     </body>
 
 
 </div>
 <script>
 
 
 $(function(){
 
     $(".info_montaria").hide();
 
     $("#input22").on("change", function(){
         if( $(this).val() == 's'){
             $(".info_montaria").show();
         }else{
             $(".info_montaria").hide();
         }
     });
 })
 
 function validate(f) {
 
     if (f.input1.value == "") {
         alert("Preencha o email!");
         return false;
     }
 
     if (f.input2.value == "") {
         alert("Preencha a Senha!");
         return false;
     }
 
     if (f.input4.value == "") {
         alert("Preencha o Apartamneto!");
         return false;
     }
 
     if (f.input5.value == "") {
         alert("Preencha o Numero!");
         return false;
     }
 
     if (f.input6.value == "") {
         alert("Preencha o Telefone!");
         return false;
     }
 
     return true;
 }
 </script>
 </body>
 </html>
